samples/bluetooth/central_hr_coded/child_image/hci_rpmsg.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
```

Comment:
Can you add `CONFIG_BT_CTLR_PHY_CODED=y`  to this configuration file?

Reply:
Yes I can. There isn't difference since `CONFIG_BT_CTLR_PHY_CODED` is enabled by default. Added

Reply:
Thanks